datumaro/components/operations.py

```diff
@@ -641,6 +656,12 @@ def _merge_mask_categories(self, sources, label_cat):
 
         return dst_mask_cat
 
+    def _merge_infos(self, sources):
+        infos = {}
+        for source in sources:
+            infos.update(source)
+        return infos
```

Comment:
Why do we need `_merge_infos()` again although we have `merge_infos()` above?

Reply:
I'll remove _merge_infos()